src/xcode/ENA/ENA.xcodeproj/project.pbxproj

```diff
@@ -5595,7 +5595,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				MARKETING_VERSION = 1.12.0;
+				MARKETING_VERSION = 1.13.0;
```

Comment:
I always thought the CI or someone later in the release process sets this label… 🤷‍♂️

Reply:
I have no idea. I accidentally create this PR for the 1.13 branch, then switched to 1.12. That switch added these changes.

Reply:
Yeah if you merged 1.13 into your branch the version change to 1.13 is in your PR as well. Please revert your changes to the *.pbxproj

Reply:
Will do. Thanks.